package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "faq-chatbot-edit",
+  "version": "1.0.0",
```

Comment:
Use version `0.0.0` for new projects. Look up Semantic Versioning or SemVer

Reply:
Changed version to 0.0.0
Notes from semantic versioning:
Given a version number MAJOR.MINOR.PATCH, increment the:
MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.